web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -0,0 +1,112 @@
+# OB/OBC Lifecycle Management Design
+Support object bucket (OB) and object bucket (OBC) CRUD
+
+## OBC
+
+### OBCs List
+The OBC list contains the following details: 
+* OBC name
+* Namespace
+* Status
+* Used
+* OBC Quota
+* Storage class
+
+The kabab menu contains the following options:
+* Edit labels
+* Edit Annotations
+* Edit OBC
+* Attach to a pod
+* Delete OBC
+
+
+![OBC-list-view](img/OBC-list.png)
+
+### Create OBC
+To create an OBC the user will need to provide the following details:
```

Comment:
Very minor detail: I think that the info icon for field level help should be next to the label, not in the field itself. (At least it was in [PF3](https://www.patternfly.org/pattern-library/forms-and-controls/help-on-forms/))

And I believe that right now we're using a gray icon in the console
![Screen Shot 2019-04-16 at 10 43 32 AM](https://user-images.githubusercontent.com/8974593/60197835-44a2e100-980e-11e9-83e4-1a6bbc0d9d05.png)